### PR TITLE
Fix how the SWF handles autoplay and preload

### PR DESCRIPTION
**Pull request by bc-bbay**
autoplay = false should not mean preload = false
preload = “auto” should mean preload

### CODE REVIEW

File: src/com/videojs/providers/HTTPVideoProvider.as [hunk 1]
```diff
@@ -278,7 +278,7 @@ package com.videojs.providers{
             _loadErrored = false;
             _loadStarted = false;
             _loadCompleted = false;
-            if (_model.preload) {
```
Comment (bclwhitaker) [inline src/com/videojs/providers/HTTPVideoProvider.as]: Should this be _model.preload()?

Comment (bc-bbay) [inline src/com/videojs/providers/HTTPVideoProvider.as]: It should not. Weird actionscript quirks.

Comment (gkatsev) [inline src/com/videojs/providers/HTTPVideoProvider.as]: getters and setters for everything is common practice, for whatever reason.

**Comment by bclwhitaker** [comment]
LGTM

**Pull request merged**